transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/LegacyFlgDestination.java

```diff
@@ -46,22 +61,43 @@ public String getName() {
   @NonNull
   @Override
   public byte[] getExtras() {
-    return encodeString(apiKey);
+    return encodeAsExtras();
   }
 
   @NonNull
-  static byte[] encodeString(@NonNull String s) {
+  public static LegacyFlgDestination parseFromExtras(@Nullable byte[] a) {
```

Comment:
Consider checking for null before calling into this method, especially given that the only time it is expected to be null is if called in the context of the `cct` backend, where the `DEFAULT_INSTANCE` might not make sense

Reply:
This was suppose to be an `EMPTY_INSTANCE`. I'll go with your suggestion and check before.